src/engraving/internal/engravingconfiguration.cpp

```diff
@@ -210,7 +210,7 @@ Color EngravingConfiguration::thumbnailBackgroundColor() const
 
 Color EngravingConfiguration::noteBackgroundColor() const
 {
-    return Color::WHITE;
+    return notationConfiguration()->foregroundColor();
```

Comment:
This solution is not ideal... It creates bad dependencies between the modules. The notation module should depend on the engraving module, and not vice versa

Reply:
How to change?

Reply:
(no such concerns had been raised with the original PR)